### PR TITLE
feat(web): add zoom toggle icon

### PR DESCRIPTION
**Pull request by mtaberna**
#2577 fast follow 

Upgrades to latest zoom-image release (0.18.2). Adds a zoom toggle icon to the asset viewer nav. We run out of horizontal space on mobile so I decided to make it desktop only.

image

### CODE REVIEW

File: web/src/lib/components/asset-viewer/photo-viewer.svelte [hunk 1]
```diff
@@ -73,15 +74,36 @@
...
+		if ($zoomImageWheelState.currentZoom == 1) {
+			setZoomImageWheelState({
+				currentZoom: 2
+			});
+		} else {
+			setZoomImageWheelState({
+				currentZoom: 1
+			});
+		}
+	};
```
Comment (uhthomas) [inline web/src/lib/components/asset-viewer/photo-viewer.svelte:87]:
```suggestion
	const doZoomImage = async () => {
	  setZoomImageWheelState({
	    currentZoom: $zoomImageWheelState.currentZoom === 1 ? 2 : 1;
	  });
	};
```

Is this nicer?

**Comment by alextran1502** [comment]
I think zooming should be enabled by default - having another button would
1. Cluster the interface
2. More clicks to perform a common action

**Comment by mtaberna** [comment]
> I think zooming should be enabled by default - having another button would
> 
> 1. Cluster the interface
> 2. More clicks to perform a common action

This feature is complementary, zooming is still enabled by default and follows other photo backup gallery approaches. 

I personally find the buttons quite useful, and I find myself using the reset functionality often to escape out of a zoomed image state. Nevertheless, I see your points and is your product call.

**Comment by alextran1502** [comment]
> > I think zooming should be enabled by default - having another button would
> > 
> > 1. Cluster the interface
> > 2. More clicks to perform a common action
> 
> This feature is complementary, zooming is still enabled by default and follows other photo backup gallery approaches.
> 
> I personally find the buttons quite useful, and I find myself using the reset functionality often to escape out of a zoomed image state. Nevertheless, I see your points and is your product call.

@mtaberna I can see the benefit of the unzoom button. Yeah we can keep this :) 

Thank you for the contribution

**Review by alextran1502 — APPROVED** [review]

**Pull request merged**